sdk/ai/azure-ai-projects/README.md

```diff
@@ -697,23 +695,6 @@ message = project_client.agents.create_message(
 
 <!-- END SNIPPET -->
 
-Azure blob storage can be used as a message attachment. In this case `VectorStoreDataSource` have to be used as a data source:
```

Comment:
Is this what Nikolay just added? Why to remove?

Reply:
Because message attachment is broken..   There is no sample to use blog store for file search as message attachment.   And I don't want to create a sample that doesn't run and associate to the readme.

Reply:
But liked I said in the sync channel, I have a fix for message attachment.
https://github.com/Azure/azure-sdk-for-python/pull/38877

Once that is merged, we can create a working sample and restore this section in readme.